src/Component/PlayersActionsContainer.js

```diff
@@ -0,0 +1,73 @@
+import React, {useState, useEffect} from 'react'
+import PlayerContainer from 'Component/PlayerContainer'
+import ActionContainer from 'Component/ActionContainer'
+import LogContainer from 'Component/LogContainer'
+
+function PlayersActionsContainer() {
+  const [started, setStarted] = useState(false)
+  const [playerhealth, setPlayerHealth] = useState(100)
+  const [monsterhealth, setMonsterHealth] = useState(120)
+  const [actionLog, setLog] = useState([])
+
+  const attack = (health, isSpecialAttack) => {
+    if (isSpecialAttack) {
+      if(playerhealth >= 90) {
+        updateLog('Player Special Attack')
+        setMonsterHealth(getMonsterHealth(health))
+      }
+    } else {
+      updateLog('Player Attack')
+      setMonsterHealth(getMonsterHealth(health))
+    }
+  }
```

Comment:
let's create 2 methods attack and special attack

Reply:
disable special attack button if it cannot be clicked

Reply:
Done